### PR TITLE
[iOS] Add upgrade banner in the FC47 Primary Course card view #460

### PR DESCRIPTION
**Pull request by forgotvas**
[iOS] Add upgrade banner in the FC47 Primary Course card view #460

Simulator Screenshot - iPhone 15 - 2024-07-11 at 17 35 42

### CODE REVIEW

**Review by rnr — APPROVED** [review]

**Comment by saeedbashir** [comment]
@forgotvas Tests are failing, could you please take a look?

**Comment by forgotvas** [comment]
> @forgotvas Tests are failing, could you please take a look?
done

File: Core/Core/CourseUpgrade/View/UpgradeCourseView.swift [hunk 1]
```diff
@@ -11,8 +11,8 @@ import Swinject
 public enum CourseAccessErrorHelperType {
     case isEndDateOld(date: Date)
     case startDateError(date: Date?)
-    case auditExpired(date: Date?, sku: String, courseID: String, pacing: String, screen: CourseUpgradeScreen)
-    case upgradeable(date: Date?, sku: String, courseID: String, pacing: String, screen: CourseUpgradeScreen)
+    case auditExpired(date: Date?, sku: String, courseID: String, pacing: String, screen: CourseUpgradeScreen, lmsPrice: Double)
+    case upgradeable(date: Date?, sku: String, courseID: String, pacing: String, screen: CourseUpgradeScreen, lmsPrice: Double)
```
Comment (saeedbashir) [inline Core/Core/CourseUpgrade/View/UpgradeCourseView.swift]: How about breaking these cases to multilines to silence the `Line Length Violation`  warning?

Comment (forgotvas) [reply Core/Core/CourseUpgrade/View/UpgradeCourseView.swift]: done

**Review by saeedbashir — APPROVED** [review]
LGTM 👍

**Comment by forgotvas** [comment]
changed trophy icon based on Moiz comment.
Simulator Screenshot - iPhone 15 - 2024-07-12 at 12 54 33

**Review by saeedbashir — APPROVED** [review]

**Pull request merged**